src/style.css

```diff
@@ -1 +1,85 @@
-/* add styles here */
+.search-bar {
+  display: flex;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  font-weight: 300;
+  position: relative;
+}
+
+.big-city::before {
+  content: '';
+  background: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  margin-left: 26px;
+  top: 25px;
+  background-size: cover;
+}
+
+.small-try::before {
+  content: '';
+  background: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  margin-left: 13px;
+  top: 15px;
+  background-size: cover;
+}
```

Comment:
these two selectors have a lot in common you can create one class common for this element add styles for big one and then override for small those you need
for instance:

```css
.label::before {
  content: '';
  background: url(images/Search.svg);
  background-repeat: no-repeat;
  background-position: center;
  position: absolute;
  height: 19px;
  width: 19px;
  margin-left: 26px;
  top: 25px;
  background-size: cover;
}

.label-small:before {
  height: 11px;
  width: 11px;
  margin-left: 13px;
  top: 15px;
}
```